tests/core/pyspec/eth2spec/test/electra/epoch_processing/test_process_pending_deposits.py

```diff
@@ -279,3 +279,42 @@ def test_processing_deposit_of_withdrawable_validator_does_not_get_churned(spec,
     # First deposit does not consume any.
     assert state.deposit_balance_to_consume == spec.get_activation_exit_churn_limit(state)
     assert state.pending_deposits == [build_pending_deposit_top_up(spec, state, validator_index=1, amount=amount)]
+
+@with_electra_and_later
+@spec_state_test
+def test_pending_deposit_over_max(spec, state):
+    # pick an amount that adds to less than churn limit
+    amount = 100 
+    overmax = spec.MAX_PENDING_DEPOSITS_PER_EPOCH_PROCESSING + 1
+    for i in range(overmax):
+        state.pending_deposits.append(spec.PendingDeposit(
+        pubkey=state.validators[i].pubkey,
+        withdrawal_credentials=state.validators[i].withdrawal_credentials,
+        amount=amount,
+        slot=state.slot
```

Comment:
```suggestion
        slot=spec.GENESIS_SLOT
```
to ensure it passes the finalized check?

Reply:
I can assert this. will look for an example